### DISCUSSION TITLE
preserve original date/timestamp of file from yt

### DISCUSSION THREAD

**Discussion by ZyroFoxtrot** [jely2002/youtube-dl-gui#404] [General]
Using the command line variant, when pulling down files from yt, the original date/timestamp of the file is preserved however using the gui variant, the default is the current time/date with no option to preserve original.

Would it be possible to bake that option in or is there a way to manipulate the current release to do so that I have not found?

Thank you

**Comment by AbeOwitz** [comment]
On my Linux box, this app seems to honor yt-dlp config files.  (probably due to embedding yt-dlp)
Create a yt-dlp config file in the appropriate place and name (per docs per your OS).  (Linux ~/.config/yt-dlp/config)
Add --mtime as an option.
Unfortunately, some videos don't match the timestamp of the YT info box.
I'm guessing it means when YT modified the video, or when the poster may have modified the video contents, title card, etc.?
This would be a nice settings option.  Even if the date is wrong, it might help with playlist order.
